librz/include/rz_util/rz_float.h

```diff
@@ -10,6 +10,30 @@
 #define RZ_FLOAT_H
 #include <rz_types.h>
 
+/**
+ *
+ * Portable float nums in C
+ */
+RZ_API float types_gen_f32_nan(void);
+RZ_API float types_gen_f32_inf(void);
+RZ_API double types_gen_f64_nan(void);
+RZ_API double types_gen_f64_inf(void);
+RZ_API long double types_gen_f128_nan(void);
+RZ_API long double types_gen_f128_inf(void);
```

Comment:
since these are `RZ_API` the name should start with `rz_`.
I would say to name them `rz_f32/64/128_inf/nan`